build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -252,29 +233,18 @@ def __str__(self):
     params = ", ".join(params)
     return f"{self.platform_type.value} device <{params}>"
 
-  def get_iree_cpu_arch_name(self,
-                             use_legacy_name: bool = False) -> Optional[str]:
+  def get_iree_cpu_arch_name(
+      self) -> Optional[common_definitions.DeviceArchitecture]:
```

Comment:
```suggestion
def get_iree_cpu_arch_name(self) -> Optional[common_definitions.DeviceArchitecture]:
```

(maybe just run autoformat)

Reply:
`yapf` actually formatted them into two lines. I shortened the method names to make them fit into a single line.

Reply:
Oh weird. I guess my column counting is off